stable/parse/values.yaml

```diff
@@ -0,0 +1,115 @@
+## Kubernetes serviceType for Parse Deployment
+## ref: http://kubernetes.io/docs/user-guide/services/#publishing-services---service-types
+##
+serviceType: LoadBalancer
+
+## loadBalancerIP for the Parse Service (optional, cloud specific)
+## ref: http://kubernetes.io/docs/user-guide/services/#type-loadbalancer
+##
+loadBalancerIP:
+
+parse:
+  ## Bitnami Parse image version
+  ## ref: https://hub.docker.com/r/bitnami/parse/tags/
+  ##
+  image: bitnami/parse:2.5.3-r0
+
+  ## Specify a imagePullPolicy
+  ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+  ##
+  imagePullPolicy: IfNotPresent
+
+  ## Parse Server Port
+  ## ref: https://github.com/bitnami/bitnami-docker-parse#configuration
+  ##
+  port: 1337
+
+  ## Parse API mount path
+  ## ref: https://github.com/bitnami/bitnami-docker-parse#configuration
+  ##
+  mountPath: /parse
+
+  ## Parse Server App ID
+  ## ref: https://github.com/bitnami/bitnami-docker-parse#configuration
+  ##
+  appId: myappID
+
+  ## Parse Server Master Key
+  ## ref: https://github.com/bitnami/bitnami-docker-parse#configuration
+  ##
+  # masterKey:
+
+  ## Configure resource requests and limits
+  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  ##
+  resources:
+    requests:
+      memory: 512Mi
+      cpu: 300m
+
+parseDashboard:
+  ## Enable deployment of Parse Dashboard
+  ##
+  enabled: true
+
+  ## Bitnami Parse Dashboard image version
+  ## ref: https://hub.docker.com/r/bitnami/parse-dashboard/tags/
+  ##
+  image: bitnami/parse-dashboard:1.1.0-r0
+
+  ## Specify a imagePullPolicy
+  ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+  ##
+  imagePullPolicy: IfNotPresent
+
+  ## Parse Dashboard application username
+  ## ref: https://github.com/bitnami/bitnami-docker-parse-dashboard#configuration
+  ##
+  username: user
+
+  ## Parse Dashboard application password
+  ## Defaults to a random 10-character alphanumeric string if not set
+  ## ref: https://github.com/bitnami/bitnami-docker-parse-dashboard#configuration
+  ##
+  # password:
+
+  ## Parse Dashboard application name
+  ## ref: https://github.com/bitnami/bitnami-docker-parse-dashboard#configuration
+  ##
+  appName: MyDashboard
+
+  ## Configure resource requests and limits
+  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  ##
+  resources:
```

Comment:
We've started recommending not to specify default resources. It's usually better to leave this a conscious choice of the user. If defaults are too high, charts may not come up successfully, especially on Minikube.